packages/dd-trace/test/appsec/waf/index.spec.js

```diff
@@ -213,39 +215,19 @@ describe('WAF Manager', () => {
         expect(ddwafContext.run).not.to.be.called
       })
 
-      it('should not call ddwafContext.run with invalid params', () => {
```

Comment:
The code is in the history so if at some point we need to recover it will be there. Besides, IMHO we should skip tests when we're sure we will recover it, in this case I'm not so sure.